commitlint.config.js

```diff
@@ -1,5 +0,0 @@
-module.exports = {
```

Comment:
Persoonlijk denk ik dat op dit soort projecten een commitlint wel nuttig kan zijn. Is die effectief verwijderd of is die op een andere manier geconfigureerd?